_includes/nav.html

```diff
@@ -13,7 +13,7 @@
 </script>
 
 <header class="header">
-    <p class="support-banner">Call <a href="tel:18666878683">1-866-OURVOTE</a> for voter support. For help using Carpool Vote, call or SMS: <a href="tel:8042393389">804-239-3389</a></p>
+    <p class="support-banner">Call <a href="tel:18666878683">1-866-OURVOTE</a> for voter support. For help using Carpool Vote, call or SMS: <a href="tel:8042393389">540-656-9388</a></p>
```

Comment:
Hey you missed the number in the URL: Please change `href="tel:8042393389"` to `href="tel:5406569388"`